libcxx/include/__split_buffer

```diff
@@ -78,23 +301,22 @@ public:
                       __split_buffer,
                       void>;
 
-  pointer __first_;
-  pointer __begin_;
-  pointer __end_;
-  _LIBCPP_COMPRESSED_PAIR(pointer, __cap_, allocator_type, __alloc_);
+  _Layout<__split_buffer> __data_;
```

Comment:
Could we make this a base instead? I think that would remove the need for a bunch of trivial wrappers throughout the class.

Reply:
Done.